example/views.py

```diff
@@ -74,8 +74,13 @@ class AuthorViewSet(ModelViewSet):
 
 
 class CommentViewSet(ModelViewSet):
-    queryset = Comment.objects.all()
+    queryset = Comment.objects.select_related('author', 'entry')
```

Comment:
Thanks for this - I merged already but will update my branch to reflect this conversation